gc/base/standard/Scavenger.cpp

```diff
@@ -510,8 +510,8 @@ MM_Scavenger::calculateRecommendedWorkingThreads(MM_EnvironmentStandard *env)
 
 	/* Calculate the average time it takes the worker threads to start collection and avgerage time workers are idle waiting for task cleanup
 	 * Calculated as (Sum_WorkerStartTime(t1 + t2 + ... + tn) - (n * collection_start)) / n  */
-	uint64_t avgTimeToStartCollection =  omrtime_hires_delta((_extensions->scavengerStats._startTime * totalThreads), _extensions->scavengerStats._workerScavengeStartTime, OMRPORT_TIME_DELTA_IN_MICROSECONDS) / totalThreads;
-	uint64_t avgTimeIdleAfterCollection =  omrtime_hires_delta(_extensions->scavengerStats._workerScavengeEndTime, (_extensions->scavengerStats._endTime * totalThreads), OMRPORT_TIME_DELTA_IN_MICROSECONDS) / totalThreads;
+	uint64_t avgTimeToStartCollection =  omrtime_hires_delta((_cycleStartTime * totalThreads), _extensions->scavengerStats._workerScavengeStartTime, OMRPORT_TIME_DELTA_IN_MICROSECONDS) / totalThreads;
+	uint64_t avgTimeIdleAfterCollection =  omrtime_hires_delta(_extensions->scavengerStats._workerScavengeEndTime, (_cycleEndTime * totalThreads), OMRPORT_TIME_DELTA_IN_MICROSECONDS) / totalThreads;
```

Comment:
Now, that _endTime is gone, I'd rename _workerScavengeEndTime to _endTime, as I originally commented. 
'Scavenge' is definitely redundant (it's already in instance/class name). 'Worker' is somewhat overly (but debatable) emphasized  (why just for these fields and not for other fields of this stats structure)?